lib/ui/platform_dispatcher.dart

```diff
@@ -283,9 +309,7 @@ class PlatformDispatcher {
   ) {
     final _ViewConfiguration previousConfiguration =
         _viewConfigurations[id] ?? const _ViewConfiguration();
-    if (!_views.containsKey(id)) {
-      _views[id] = FlutterView._(id, this);
-    }
+    _views.putIfAbsent(id, () => _createView(id));
```

Comment:
We should probably remove this? You shouldn't be able to update metrics for non-existing views.